### DISCUSSION TITLE
Quick way to view data

### DISCUSSION THREAD

**Discussion by mdav43** [aaronpk/Overland-iOS#119] [Show and tell]
Hi Aaron - just wanted to say thanks for the app.

Wanted to share a quick way to view some of the data. I've previously shared a simple python script which can be used to save the app data down to a local folder - so the below assumes you have a folder with all your geojson files.

**Steps**
1. Use this script to combine geojson files.
```
jq -r '
  [["timestamp","lat","long","motion"]],
   [ .locations[]
     | [ .properties.timestamp, .geometry.coordinates[0], .geometry.coordinates[1],( try (.properties.motion[] | tostring) ) // "null" ]
    ]
  | .[]
  | @csv' *.geojson >> exportcsv.csv
```

 2. Upload to [https://kepler.gl/demo](https://kepler.gl/demo) - an Uber framework. You can do playbacks and lots of other stuff. I believe the data is actually all stored locally in your browser - not uploaded to Uber.

Attached is a demo of what can be quickly seen in under 2 mins.

https://user-images.githubusercontent.com/3327386/167785791-a935f437-0eab-4f33-8a36-0e59c5d986c8.mov

Note: Given the low visibility in the preview, I'm ok with the privacy.

**Comment by aaronpk** [comment]
This is very cool! Thanks for sharing!
